BlazorBarcodeScanner.ZXing.JS/wwwroot/BlazorBarcodeScanner.js

```diff
@@ -182,26 +182,17 @@ window.BlazorBarcodeScanner = {
             mediaStreamSetTorch(track, torchStatus);
         }
     },
-    capture: async function (type, canvas) {
+    capture: async function (type, canvas,video) {
         this.lastPicture = "";
 
         if (!this.codeReader.stream) {
             return "";
         }
 
-        let capture = new ImageCapture(this.codeReader.stream.getVideoTracks()[0]);
-
-        await capture.grabFrame()
-            .then(bitmap => {
-                let context = canvas.getContext('2d');
-
-                canvas.width = bitmap.width;
-                canvas.height = bitmap.height;
-
-                context.drawImage(bitmap, 0, 0, bitmap.width, bitmap.height);
-
-                this.lastPicture = canvas.toDataURL(type);
-            });
+        canvas.width = video.videoWidth;
```

Comment:
Style changes are good, but this is another concern we need to address in another PR, also keep ImageCapture and if video parameter passed use it otherwise rely on ImageCapture path.